mailpoet/lib/Config/Menu.php

```diff
@@ -283,6 +282,13 @@ private function registerMailPoetSubMenuEntries() {
       ]
     );
 
+    // add body class for form editor page
```

Comment:
Is this comment correct? I think it should be related to the email editor.

Reply:
Ahh. I copy pasted the code for adding the class and forgot to update it. Nice catch!